src/main/java/core/basesyntax/Figures.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class Figures implements Area, Drawable {
+    private String color;
+
+    public Figures(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public abstract double getArea();
+
+    public abstract void draw();
```

Comment:
There is no need to mention methods of interface in abstract class if they remain abstract. Override in children classes is enough
```suggestion
```